hack/tools/swagger/lib/render.go

```diff
@@ -85,6 +85,7 @@ func RenderOpenAPISpec(cfg Config) (string, error) {
 		return "", err
 	}
 	serverConfig.OpenAPIConfig = genericapiserver.DefaultOpenAPIConfig(cfg.GetOpenAPIDefinitions, openapi.NewDefinitionNamer(cfg.Scheme))
+	serverConfig.OpenAPIV3Config = genericapiserver.DefaultOpenAPIV3Config(cfg.GetOpenAPIDefinitions, openapi.NewDefinitionNamer(cfg.Scheme))
```

Comment:
This change is for adopting Kubernetes #114998, please @jwcesign @lonelyCZ help to take a look. Otherwise the error([from here](https://github.com/kubernetes/kubernetes/blame/ad9b60e2c9ddb21e8b00cabbe27e639638a0ea88/staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go#L966)) would be:
```
-bash-5.0# hack/update-swagger-docs.sh 
W1127 12:00:49.368322 3935808 recommended.go:152] Neither kubeconfig is provided nor service-account is mounted, so APIPriorityAndFairness will be disabled
W1127 12:00:49.369498 3935808 authentication.go:52] Authentication is disabled
F1127 12:00:49.371656 3935808 render.go:129] unable to get openapi models: OpenAPIV3 config must not be nil
```

Reply:
I tried this PR, and it works fine:
![image](https://github.com/karmada-io/karmada/assets/19745536/a7ad5e3f-8aef-44ba-9563-22135ec57ea3)

Reply:
Thanks.